tests/v3/test_store/test_zip.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+import os
+import tempfile
+from collections.abc import Coroutine
+from typing import Any
+
+import pytest
+
+from zarr.abc.store import AccessMode
+from zarr.core.buffer import Buffer, default_buffer_prototype
+from zarr.store.zip import ZipStore
+from zarr.testing.store import StoreTests
+
+
+class TestZipStore(StoreTests[ZipStore]):
```

Comment:
Just a note that we should have nice docstrings for the store classes, and their methods. This is not a blocker, and can be handled by a later PR.